sync/v1.proto

```diff
@@ -275,6 +275,26 @@ message EntitlementInfo {
   repeated Entitlement entitlements = 2;
 }
 
+// Status of the analysis of the leaf signing certificate
+enum SigningStatus {
+  SIGNING_STATUS_UNSPECIFIED = 0;
+
+  // The executing binary wasn't signed'
+  SIGNING_STATUS_UNSIGNED = 1;
+
+  // There was a problem checking the validity of the code signature of the executing binary
```

Comment:
This comment makes it sound like we failed to check the signature rather than the signature was invalid. IIRC it's actually either case, maybe the comment could be clearer?